frontend/package.json

```diff
@@ -0,0 +1,98 @@
+      "dev:next": "next dev",
+      "dev:codegen": "DOTENV_CONFIG_PATH=./.env.development graphql-codegen -r dotenv/config --config codegen.yml --watch",
+      "build": "npm run build:codegen && npm run build:next",
+      "build:next": "next build",
+      "build:codegen": "DOTENV_CONFIG_PATH=./.env.production graphql-codegen -r dotenv/config --config codegen.yml",
+      "start": "next start",
+      "type-check": "tsc --pretty --noEmit",
+      "format": "prettier --write .",
+      "lint": "eslint . --ext ts --ext tsx --ext js",
+      "test": "jest",
+      "test-all": "npm run lint && npm run type-check && npm run test",
+      "cypress:open": "cypress open",
+      "cypress:run": "cypress run",
+      "transform-svg": "npx @svgr/cli --typescript -d assets/svg assets/svg/files && prettier --write assets/svg"
+   },
+   "husky": {
+      "hooks": {
+         "pre-commit": "lint-staged",
+         "pre-push": "npm run type-check"
+      }
+   },
+   "lint-staged": {
+      "*.@(ts|tsx)": [
+         "npm run lint",
+         "npm run format"
+      ]
+   },
+   "dependencies": {
+      "@chakra-ui/react": "1.6.3",
+      "@emotion/react": "11.1.5",
+      "@emotion/styled": "11.3.0",
+      "@ifixit/react-components": "0.3.0",
+      "@xstate/fsm": "1.6.1",
+      "@xstate/react": "1.5.1",
+      "algoliasearch": "4.9.1",
+      "dayjs": "1.10.5",
+      "framer-motion": "4.1.11",
+      "graphql": "15.5.3",
+      "immer": "9.0.2",
+      "lodash": "4.17.21",
+      "next": "11.0.1",
+      "query-string": "7.0.1",
+      "react": "17.0.2",
+      "react-dom": "17.0.2",
+      "react-fast-compare": "3.2.0",
+      "react-icons": "4.2.0",
+      "react-virtualized-auto-sizer": "1.0.5",
+      "react-window": "1.8.6",
+      "snarkdown": "2.0.0"
+   },
+   "devDependencies": {
+      "@graphql-codegen/cli": "2.1.1",
+      "@graphql-codegen/typescript": "2.2.0",
+      "@graphql-codegen/typescript-generic-sdk": "2.1.3",
+      "@graphql-codegen/typescript-operations": "2.1.3",
+      "@svgr/cli": "5.5.0",
+      "@testing-library/cypress": "7.0.6",
+      "@testing-library/jest-dom": "5.13.0",
+      "@testing-library/react": "11.2.5",
+      "@testing-library/user-event": "13.1.9",
+      "@types/jest": "26.0.20",
+      "@types/node": "14.14.25",
+      "@types/react": "17.0.1",
+      "@types/react-virtualized-auto-sizer": "1.0.1",
+      "@types/react-window": "1.8.4",
+      "@types/shopify-buy": "2.10.5",
+      "@typescript-eslint/eslint-plugin": "4.14.2",
+      "@typescript-eslint/parser": "4.14.2",
+      "babel-jest": "26.6.3",
+      "concurrently": "6.2.1",
+      "cypress": "7.5.0",
+      "dotenv": "10.0.0",
+      "eslint": "7.19.0",
+      "eslint-config-next": "^11.0.1",
+      "eslint-config-prettier": "7.2.0",
+      "eslint-plugin-cypress": "2.11.3",
+      "eslint-plugin-react": "7.19.0",
+      "eslint-plugin-react-hooks": "4.2.0",
+      "husky": "4.2.3",
```

Comment:
Curious why version 4 is being used for Husky instead of version 7?

Reply:
@ardelato Hard to remember as this was done at the beginning of the project.. I think it was due to some breaking change with other tooling.. We can try to updated it later and see if it breaks something